src/components/ImageView/Toolbar/ToolbarComponent.tsx

```diff
@@ -35,6 +35,10 @@ export class ToolbarComponent extends React.Component<ToolbarComponentProps> {
         this.props.appStore.activeFrame.regionSet.setMode(RegionMode.CREATING);
     };
 
+    handleCoordinateSystemClicked = (coordinateSystem: string) => {
+        this.props.appStore.overlayStore.global.setSystem(coordinateSystem as SystemType);
```

Comment:
this breaks when passing "Native" as the coordinate type, because AST expects an empty string, instead of"Native". 

We should probably also migrate to "Auto" instead of "Native", to be consistent with the rest of the UI's language

Reply:
Fixed